src/runner.ts

```diff
@@ -147,41 +147,44 @@ export class TestsRunner {
     }
 
     private findParasoftXmlReport(report: string, workingDir: string) : string | undefined {
```

Comment:
This is only a compliment:
I like how you refactored the structure of the logic in this method which makes the logic more straightforward.